### ISSUE TITLE
Issue in funnel run query

### ISSUE THREAD

**Issue by Maxime-J** [umami-software/umami#2123]
### Describe the Bug

Testing with MySQL, there's an error when I run a funnel report query.
No matter what params I put, weither a match is expected or not, the request doesn't succeed.

I didn't manage to see what's causing the issue for now.
Prisma error isn't very helpful...
and generated query doesn't fail when run through phpMyAdmin.

### Database

MySQL

### Relevant log output

```shell
Invalid `prisma.$queryRawUnsafe()` invocation:
Raw query failed. Code: `N/A`. Message: `N/A`
```

### How are you deploying your application? (if relevant)

Test VM

**Comment by franciscao633** [comment]
Confirmed bug. We are looking into this.

**Comment by mikecao** [comment]
Fixed in v2.4.0.
